support-frontend/assets/stylesheets/skeleton/webfonts.scss

```diff
@@ -2,7 +2,7 @@ $gu-gt-fonts-url: "//pasteup.guim.co.uk/webfonts/1.0.0";
 $gu-gt-font-variant: "noalts-not-hinted";
```

Comment:
can we rename this file to `non-critical-webdonts.scss` or something and add a comment to clarify most webfonts aren't here?